dangerfile.js

```diff
@@ -0,0 +1,31 @@
+// eslint-disable-next-line import/no-extraneous-dependencies, no-unused-vars
```

Comment:
Nope, with the token set up in travis CI and this file, everything is set up to start executing this file when travis CI runs.